gradle/generate-defaults.gradle

```diff
@@ -42,6 +42,7 @@ configure(rootProject) {
                 "systemProp.file.encoding=UTF-8",
                 "org.gradle.jvmargs=-Xmx3g", // TODO figure out why "gradlew check" runs out of memory if 2g
                 "org.gradle.parallel=true",
+                "org.gradle.caching=true",
```

Comment:
This enables local caches, right? I'm not sure folks will be enthusiastic about their home folders increasing in size so dramatically - solr build outputs are pretty heavy.

Reply:
Right, so if you're afraid of this, I can remove this setting and you can enable it through the command line when building, check after a few weeks of dev if it's acceptable. It's hard to predict the size, currently I see on my machine it's about 400Mb (after removing the cache directory and running a few solr builds)

Reply:
I think this should be left commented out in the generated user defaults - let it be an opt-in. I plan to commit gradle 7.2 path tomorrow and reiterate on your patch, cherry picking the obvious parts of it. Stay tuned. :)